services/core-api/app/__init__.py

```diff
@@ -70,9 +70,9 @@ def log_response_info(response):
         path = request.path
         ip_address = request.remote_addr
         http_version = request.environ.get('SERVER_PROTOCOL', 'HTTP/1.1')
-
-        # Log combined request and response information
-        current_app.logger.info(f'{ip_address} - - [{get_formatted_current_time()}] "{method} {path} {http_version}" {response.status_code} -')
+        if path != '/health':
+            # Log combined request and response information
+            current_app.logger.info(f'{ip_address} - - [{get_formatted_current_time()}] "{method} {path} {http_version}" {response.status_code} -')
 
```

Comment:
🙌 🙌 🙌 

This has been driving me bonkers, really should have dealt with it before! Thanks!